core/src/components/toast/toast.tsx

```diff
@@ -34,6 +34,7 @@ import type { ToastButton, ToastPosition, ToastLayout } from './toast-interface'
  * @virtualProp {"ios" | "md"} mode - The mode determines which platform styles to use.
  *
  * @part button - Any button element that is displayed inside of the toast.
+ * @part button-cancel - Any button element with role "cancel" that is displayed inside of the toast.
```

Comment:
Same as my other comment: Let's call this `button cancel`
```suggestion
 * @part button cancel - Any button element with role "cancel" that is displayed inside of the toast.
```